atmo/jobs/forms.py

```diff
@@ -221,10 +221,13 @@ def clean_start_date(self):
     def save(self, commit=True):
         obj = super(EditSparkJobForm, self).save(commit=False)
         if 'start_date' in self.changed_data:
```

Comment:
@robhudon Totally, apologies, you're right, we can fetch that info in the future with `spark_job.runs.datetimes('run_date', 'second', order='DESC')[0]` or `spark_job.runs.latest().run_date`.